doc/tech/glossary.md

```diff
@@ -96,7 +96,7 @@ A **Container** provides operating-system-level virtualisation by abstracting th
 - **Container Engine**: Software components used to create, destroy, and manage containers on top of an operating system.
 
 - **Pod**:	The smallest and simplest Kubernetes object. A Pod represents a set of running containers on your cluster. A Pod is typically set up to run a single primary container. It can also run optional sidecar containers that add supplementary features like logging.
-- **Kubernetes Cluster**: A set of machines, called nodes and master, that run containerised applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
+- **Kubernetes Cluster**: A set of machines, called nodes and master, that run container-based and VM-based applications managed by Kubernetes. A cluster has at least one worker node and at least one master.
```

Comment:
Kubernetes uses "containerized" in the cluster definition.

Reply:
Ah yes good spot - will reverse this.

Reply:
Corrected here: https://github.com/cntt-n/CNTT/pull/1652/commits/ef5b3eac5197a6429ee6a334e594123170c64776